frontend/lib/ui/privacy-info-modal.tsx

```diff
@@ -56,14 +56,14 @@ export const TermsOfUseLink: React.FC<{ text?: string }> = ({ text }) => (
 export function PrivacyInfoModal(props: {}): JSX.Element {
   return (
     <Modal
-      title={li18n._(t`Your privacy is very important to us!`)}
+      title={li18n._(t`Your privacy is very important to us`)}
       onCloseGoTo={BackOrUpOneDirLevel}
       render={(ctx) => (
         <>
           <div className="jf-is-scrollable-if-too-tall">
             <Trans id="justfix.privacyInfoModalText">
```

Comment:
You may need to update this id here to get rid of the old translation? maybe call it `justfix.privacyInfoModalTextv2`?